app/models/comfy/cms/site.rb

```diff
@@ -36,7 +38,9 @@ def self.find_site(host, path = nil)
     cms_site = nil
 
     public_cms_path = ComfortableMexicanSofa.configuration.public_cms_path
-    path.gsub!(%r{\A#{public_cms_path}}, "") unless path.nil? || public_cms_path == "/"
+    unless path.nil? || public_cms_path == "/"
```

Comment:
Rubocop didn't complain about `unless expession || expression`? Should be flipped to `if` for readability. My brain short-circuits on these.

Reply:
Done.